src/protocol.cpp

```diff
@@ -1618,11 +1633,11 @@ bool CProtocol::EvaluateCLServerListMes ( const CHostAddress&     InetAddr,
 
         // IP address (4 bytes)
         const quint32 iIpAddr =
-            static_cast<int> ( GetValFromStream ( vecData, iPos, 4 ) );
+            static_cast<uint32_t> ( GetValFromStream ( vecData, iPos, 4 ) );
```

Comment:
Why not cast to "quint32"?

Reply:
Same question applies to other places in the code, too.

Reply:
I'll go back over the ones I changed to check what the variable is used for and make sure the declaration and cast both use that target data type.

Reply:
Done
91b9c04 Resolve casts to target types